python/modules/IcePy/Slice.cpp

```diff
@@ -147,6 +148,9 @@ IcePy_loadSlice(PyObject* /*self*/, PyObject* args)
             }
         }
 
+        PyObject* builtins = PyEval_GetBuiltins();
+        PyDict_SetItemString(moduleDict, "__builtins__", builtins);
```

Comment:
Without this importing modules can result in errors. When importing from outside a package.